### PR TITLE
Added video and some other minor updates

### PR DESCRIPTION
**Pull request by hroyrh**
Added demo video to compliment the blog post and a few other minor updates.

### CODE REVIEW

File: _posts/2020-06-18-Metal3-dev-env-BareMetal-Cluster-Deployment.md [hunk 1]
```diff
@@ -371,10 +373,16 @@ sh ${metal3-dev-env-path}/scripts/deprovision/worker.sh
 sh ${metal3-dev-env-path}/scripts/deprovision/controlplane.sh
 sh ${metal3-dev-env-path}/scripts/deprovision/cluster.sh
 ```
-> Note :
+> note "Note"
 > The reason for running the `deprovision/worker.sh` and `deprovision/controlplane.sh` scripts is that not all objects are cleared when we just run the `deprovision/cluster.sh` script. Following this, if you want to deprovision control plane it is recommended to deprovision the cluster itself since we can't provision a new control plane with the same cluster. For worker deprovisioning, we only need to run the worker script.
 
 
+
+This video below exhibits the provisioning and deprovisioning a Kubernetes Cluster as shown above.
```
Comment (codificat) [inline _posts/2020-06-18-Metal3-dev-env-BareMetal-Cluster-Deployment.md:381]: Maybe it's just my non-native English, but I think this sentence is a bit unclear.

An alternative proposal:

> The following video shows all the steps to provision and de-provision a Kubernetes cluster explained above:

But if you prefer the current version (use your judgement :wink: ) I believe it needs at least an _of_: "...exhibits the provisioning ... *of* a Kubernetes...")

**Review by codificat — CHANGES_REQUESTED** [review]
Looks good, great improvement with the admonitions.

Just a minor clarification in the introduction of the video.

**Comment by codificat** [comment]
Besides the comments on your update, tests are now failing due to a broken link in an older blog post.

I think the simplest/quickest fix is if you can just add another commit here to address that: it's in ` _posts/2019-06-25-Metal3.md` and the link to change is:

https://github.com/metal3-io/baremetal-operator/blob/master/pkg/apis/metal3/v1alpha1/baremetalhost_types.go

We can change the link to reference a tag to keep it working:
https://github.com/metal3-io/baremetal-operator/blob/capm3-v0.3.0/pkg/apis/metal3/v1alpha1/baremetalhost_types.go

File: _posts/2020-06-18-Metal3-dev-env-BareMetal-Cluster-Deployment.md (hunk 1 above)
Comment (hroyrh) [reply _posts/2020-06-18-Metal3-dev-env-BareMetal-Cluster-Deployment.md]: Thanks for the review!
Fixed, let me know if its looks okay now.

**Pull request merged**